Tracking/src/Tracking/Reco/SeedFinderProcessor.cxx

```diff
@@ -575,7 +580,8 @@ void SeedFinderProcessor::FindSeedsFromMap(ldmx::Tracks& seeds,
       // A seed is rejected if it is found incompatible with all the target
       // extrapolations
 
-      bool tgt_compatible = false;
+      // This is set but unused, FIXME
+      // bool tgt_compatible = false;
```

Comment:
@#@bloodyyugo  can I delete this?
```suggestion
```

Reply:
Keep this and related code...eventually we will use tagger track position at target to inform recoil tracking.